lib/themes/dosomething/paraneue_dosomething/scss/patterns/_photo.scss

```diff
@@ -0,0 +1,29 @@
+.photo {
+
+  &.-framed {
+    background-color: #fff;
+    padding: ($base-spacing / 4);
+
+    @include media($larger) {
+      padding: ($base-spacing / 2);
+    }
+
+    img {
+      width: 100%;
+    }
+
+    .__copy {
```

Comment:
Doesn't have to be in this PR, but can you update this to `.photo__copy` sometime?